### DISCUSSION TITLE
Test FedCM for BYOIDP

### DISCUSSION THREAD

**Discussion by erlend-sh** [goauthentik/authentik#9350] [Ideas]
For advocates of digital identity ownership, right now there’s a unique opportunity to tip the scales in favor of the independent web:

https://wrily.foad.me.uk/sign-in-with-big-tech-only-or-sign-in-with-whom-i-prefer

The FedCM drafters have been open to reference implementations for a month: https://github.com/fedidcg/FedCM/issues/240#issuecomment-2004650817

Apparently there’s not even a need to use Chrome Canary for testing any longer, as an experimental option for FedCM has been added to main Chrome:

<img width="819" alt="image" src="https://github.com/portier/portier-broker/assets/583842/d2f1a3a7-4c00-4d91-9f4b-d16d678e629c">

So far there’s one nearly [working example in Rauthy](https://github.com/sebadob/rauthy/discussions/145#discussioncomment-8831943).

With a few more proofs-of-concept, we can ensure that this emerging browser standard will benefit the entirety of the open web. Your assistance would be hugely appreciated by your fellow indies! 🙏

**Comment by BeryJu** [comment]
Thanks for the idea and we'll keep it in mind as it and authentik evolves.
